src/vast_tracker.js

```diff
@@ -605,11 +604,8 @@ export class VASTTracker extends EventEmitter {
       ) {
         macros['ASSETURI'] = this.creative.mediaFiles[0].fileURL;
       }
-      if (!macros['CONTENTPLAYHEAD'] && this.progress) {
-        //CONTENTPLAYHEAD @deprecated in VAST 4.1 replaced by ADPLAYHEAD & CONTENTPLAYHEAD
```

Comment:
We should not remove support for the CONTENTPLAYHEAD macro because if we do, we will not be backwards compatible as we will not follow the IAB specification for VAST pre 4.1.